interface/reports/dictation_review.php

```diff
@@ -0,0 +1,316 @@
+                                <td class='control-label'>
+                                    <input type='text' name='form_from_date' id="form_from_date"
+                                           class='datepicker form-control'
+                                           size='10' value='<?php echo attr($from_date) ?>'
+                                           title='yyyy-mm-dd'>
+                                </td>
+                                <td class='control-label'>
+                                    <?php xl('To','e'); ?>:
+                                </td>
+                                <td>
+                                    <input type='text' name='form_to_date' id="form_to_date"
+                                           class='datepicker form-control'
+                                           size='10' value='<?php echo attr($to_date) ?>'
+                                           title='yyyy-mm-dd'>
+                                </td>
+                            </tr>
+                        </table>
+
+                    </div>
+
+                </td>
+                <td align='left' valign='middle' height="100%">
+                    <table style='border-left:1px solid; width:100%; height:100%' >
+                        <tr>
+                            <td>
+                                <div class="text-center">
+                                    <div class="btn-group" role="group">
+                                    <a href='#' class='btn btn-default btn-save' onclick='$("#form_refresh").attr("value","true"); $("#theform").submit();'>
+					<span>
+						<?php xl('Submit','e'); ?>
+					</span>
+                                    </a>
+
+                                    <?php if ($_POST['form_refresh']) { ?>
+                                        <a href='#' class='css_button' onclick='window.print()'>
+						<span>
+							<?php xl('Print','e'); ?>
+						</span>
+                                        </a>
+                                        <a href='#' class='css_button' onclick='$("#form_refresh1").attr("value","true"); $("#theform").submit();'>
+					<span>
+						<?php xl('Save','e'); ?>
+					</span>
+                                        </a>
+                                    <?php } ?>
+                                </div>
+                              </div>
+                            </td>
+                        </tr>
+                    </table>
+                </td>
+            </tr>
+        </table>
+    </div> <!-- end of parameters -->
+
+    <?php
+    /**
+     * This is to record to the database with the posted information from the Save button being clicked.
+     */
+    if ($_POST['form_refresh1'] == true){
+
+
+        foreach ($_POST as $key => $val){        //note: next steps are to integrate this with the function below to complete the save
+            if (preg_match("/^signed/", $key)){
+                echo $val ."<br>";
```

Comment:
escape with text()
``````
echo text($val) ."<br>";

Reply:
I removed this all together, it was for testing a theory.